src/main.cpp

```diff
@@ -4431,7 +4421,7 @@ void GridcoinServices()
             {
                 if (fDebug) printf("SVC: set off Tally (v3 B) height %d\n",nBestHeight);
                 if (fDebug10) printf("#TIB# ");
-                bDoTally_retired = true;
+                TallyResearchAverages(pindexBest);
```

Comment:
I am considering removing these two tallies. The previous calls were not busy waiting so the previous behavior was undefined.

Reply:
it is complicated